.github/actions/github-release/README.md

```diff
@@ -10,7 +10,7 @@ perform github releases but they all tend to have their set of drawbacks.
 Additionally nothing handles deleting releases which we need for our rolling
 `dev` release.
 
-To handle all this this action rolls-its-own implementation using the
```

Comment:
This sentence was correct before and sounds broken now. It needs to have "this" two times, one for the phrase, and one as an adjective of the subject. However, it probably should have had a comma before between them.